tests/src/test/scala/org/http4s/ServerSentEventSpec.scala

```diff
@@ -117,6 +121,11 @@ class ServerSentEventSpec extends Http4sSuite {
         .through(ServerSentEvent.decoder)
         .compile
         .toVector
+      val f = roundTrip.unsafeToFuture()
+      val rt = Await.result(f, Duration(10, TimeUnit.SECONDS))
+      if ( rt != sses ) {
+        toString
+      }
```

Comment:
I'm confused what's going on here, particularly with the blocking.  Is this for a better failure message?

Reply:
Ooof I will remove that is me hacking in something for debugging...

Reply:
That code should have been removed.